packages/fast-components-styles-msft/src/patterns/input-field.ts

```diff
@@ -0,0 +1,43 @@
+import { applyTypeRampConfig } from "../utilities/typography";
+import { fontWeight } from "../utilities/fonts";
+import outlinePattern from "../patterns/outline";
+import typographyPattern from "../patterns/typography";
+import { CSSRules } from "@microsoft/fast-jss-manager";
+import { DesignSystem } from "src/design-system";
```

Comment:
Is this resolving?

Reply:
It works but I updated

Reply:
thx - I think it's possible that this would've thrown once packaged up and exported.